flixel/addons/editors/tiled/TiledImageSet.hx

```diff
@@ -0,0 +1,51 @@
+package flixel.addons.editors.tiled;
+
+import haxe.xml.Fast;
+
+/**
+ * Copyright (c) 2013 by Samuel Batista
+ * (original by Matt Tuttle based on Thomas Jahn's. Haxe port by Adrien Fischer)
+ * This content is released under the MIT License.
+ */
+class TiledImageSet implements Dynamic<String>
```

Comment:
This is a property:
 <tile id="0">
   <properties>
    <property name="type" value="player"/>
   </properties>
  </tile>

This is an image:
 <tile id="0">
   <image width="769" height="720" source="../images/backgrounds/pack2/capa01_A.png"/>
  </tile>

Properties only have two parameters and image has three.